platform/admin/ui/Gruntfile.js

```diff
@@ -89,14 +87,48 @@ module.exports = function (grunt) {
                 tasks: ['bower']
             }
         },
-        casperjs: {
+        mochaWebdriver: {
             options: {
-                async: {
-                    parallel: false
+                autoInstall: true,
+                usePromises: true,
+                reporter: 'spec',
+                timeout: 1000 * 30,
+                slow: 10000
+            },
+            phantom: {
+                src: ['src/test/js/wd/*.js'],
+                options: {
+                    hostname: '127.0.0.1',
+                    usePhantom: true,
+                    phantomPort: 5555
+                }
+            },
+            selenium: {
+                src: ['src/test/js/wd/*.js'],
+                options: {
+                    // make sure to start selenium server at host:port first
+                    hostname: '127.0.0.1',
+                    port: 4444,
+                    // mochaWebdriver always starts a selenium server so
+                    // starting phantomjs instance that will not be used
+                    phantomPort: 5555,
+                    usePhantom: true
                 }
             },
-            //this is where the tests would be called from
-            files: ['src/test/js/*.js']
+            sauce: {
```

Comment:
Did you try running these tests against Sauce Labs and verify that it worked with all three of these browsers on Windows?